filters/include/pcl/filters/impl/crop_hull.hpp

```diff
@@ -136,12 +135,12 @@ pcl::CropHull<PointT>::applyFilter2D (PointCloud &output)
           output.push_back (input_->points[(*indices_)[index]]);
         // once a point has tested +ve for being inside one polygon, we can
         // stop checking the others:
-        break;
+        return;
```

Comment:
No, but you are right. That's an error. Just saw `poly == hull_polygons_.size ()` will be only true, in case the `break` was never called. And in case `break` is called, nothing further will be done. Didn't saw the outer loop, so `return` isn't correct.